src/introduction/prerequisites.md

```diff
@@ -26,13 +26,13 @@ of the following options for your operating system:
 - Windows
 
 ```{note}
-If you are using Windows, you will have to install the Windows Subsystem for Linux 2.
+If you are using Windows, you will have to install the [Windows Subsystem for Linux 2](https://learn.microsoft.com/en-us/windows/wsl/install) (WSL2).
 Visit the `cwltool` [documentation](https://github.com/common-workflow-language/cwltool/blob/main/README.rst#ms-windows-users)
 for details on installing WSL2.
```

Comment:
Best not to link directly to the WSL2 install docs, as users will click that first and not follow all of the linked cwltool MS Windows installation directions

```suggestion
If you are using Windows, you will have to install the Windows Subsystem for Linux 2 as documented in the
[`cwltool` documentation for Microsoft Windows users](https://github.com/common-workflow-language/cwltool/blob/main/README.rst#ms-windows-users).
```